### ISSUE TITLE
Grunt 0.4.0rc5 issues

### ISSUE THREAD

**Issue by vkamenski** [ehynds/grunt-image-embed#11]
Hi,

I've got error with grunt v0.4.0rc5 and v0.1.3 of the plugin

Loading "grunt-image-embed.js" tasks...ERROR
TypeError: Cannot read property '_' of undefined

As far as I understand this is compatibility issue and v0.1.3 doesn't support grunt v0.4.0. 
Am I right?

**Comment by eskimoblood** [comment]
Seems the actual version isn't updated on npm, if you use the it directly from github it works:

```
"devDependencies": {
    "grunt-image-embed": "https://github.com/ehynds/grunt-image-embed/archive/master.tar.gz"
  },

```

**Comment by ehynds** [comment]
I just published v0.2.0 to npm which does include grunt 4.0 support. Sorry for the delay on that!
